cypress/page/menu-content.page.ts

```diff
@@ -2,7 +2,7 @@ export class MenuContentPage {
   private tShirtMenuSelector: string;
 
   constructor () {
-    this.tShirtMenuSelector = '#block_top_menu > ul > li:nth-child(3) > a';
+    this.tShirtMenuSelector = '#block_top_menu > ul > li > [title="T-shirts"]';
```

Comment:
i think you can improve this with https://docs.cypress.io/api/commands/within.html#Syntax